Samples/WidgetAdvSample/WidgetAdvSample.vcxproj

```diff
@@ -45,7 +45,7 @@
   </ItemGroup>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>

```

Comment:
Do we want to keep this at v142 to support VS2019?

Reply:
Yes, I went to update that, but then saw all the jobs have logic to switch to the appropriate version based on the VS you're using. So this should be fine.